src/main/java/net/mcreator/ui/component/JMinMaxSpinner.java

```diff
@@ -86,6 +108,14 @@ public double getMaxValue() {
 		return (double) max.getValue();
 	}
 
+	public int getIntMinValue() {
+		return getMinNumber().intValue();
+	}
+
+	public int getIntMaxValue() {
+		return getMaxNumber().intValue();
+	}
+
 	public void setMinValue(double val) {

```

Comment:
>You should also consider that existing workspaces may use invalid values (where min > max) so setters need to take this into the account.

Reply:
Or does the change listener handle this implicitly (tested?)?

Reply:
I've tried setting my test custom block to generate between minimum of Y=64 and maximum of Y=32. Opening this block on this branch caused the minimum height to get offset to be equal to the maximal height.